lib/cylc/cfgspec/globalcfg.py

```diff
@@ -189,12 +191,13 @@
             'global init-script': vdr(vtype='string'),
             'copyable environment variables': vdr(
                 vtype='string_list', default=[]),
-            'retrieve job logs': vdr(vtype='boolean', default=False),
+            'retrieve job logs': vdr(vtype='boolean'),
```

Comment:
I think the default was put in under `__MANY__` by a calculated mistake. The logic automatically default to use settings under `localhost`, which normally has all the defaults.